### ISSUE TITLE
Problem using env in values.yaml

### ISSUE THREAD

**Issue by EE041B3-UnionInvestment** [SonarSource/helm-chart-sonarqube#161]
https://github.com/SonarSource/helm-chart-sonarqube/blob/be548a03ce18bf302405e258a187a66822da6974/charts/sonarqube-lts/templates/sonarqube-sts.yaml#L242

does this belong to "resources:"? I think this prevents me from using

env:
  - name x
     value: y

**Comment by tobias-trabelsi-sonarsource** [comment]
Hi @EE041B3-UnionInvestment :wave: 

what is the issue that you are facing? The statement is not under `ressources`. The complete template statement looks like this:

```go
{{- with .Values.env }}
env:
{{- . | toYaml | trim | nindent 12 }}
{{- end }}
```

which gets rendered to something like this if env values are defined:

```yaml
env:
  - name: VARIABLE
    value: my-value
```
prettifying the indentation in the example of cause. if ` .Values.env` is `nil` (the default), the whole section does not get rendered.

**Comment by EE041B3-UnionInvestment** [comment]
Hi, thanks for your reply. I added

```yaml
env:
  - name: fubar
    value: barfu
```

to my values file but trying to render it fails:

`Error: YAML parse error on sonarqube-lts/templates/sonarqube-sts.yaml: error converting YAML to JSON: yaml: line 98: did not find expected key
helm.go:88: [debug] error converting YAML to JSON: yaml: line 98: did not find expected key`


<details>

```yaml
ingress:
  annotations:
    appgw.ingress.kubernetes.io/ssl-redirect: "true"
    appgw.ingress.kubernetes.io/use-private-ip: "false"
    kubernetes.io/ingress.class: azure/application-gateway
  enabled: true
  hosts:
    - name: redacted
      path: /
  tls:
   - secretName: external-wildcard-tls
     hosts:
       - sonarqube.test.cloudplatform.union-investment.de
prometheusExporter:
  enabled: false
jdbcUrlOverride: "jdbc:postgresql://redacted:5432/sonarqube?user=postgresqladmin@redacted&sslmode=require"

postgresql:
  enabled: false
  existingSecret: "postgresql-admin-password"
  existingSecretPasswordKey: "postgresql-password"

plugins:
  install:
    - "https://redacted/sonarqube/sonar-auth-aad-plugin-1.2.0.jar"

persistence:
  enabled: true
  existingClaim: "pvc-sonarqube-data"

env:
  - name: fubar
    value: barfu
```
</details>

**Comment by tobias-trabelsi-sonarsource** [comment]
Ah gotcha! This has been fixed with #134 but has not been released yet. you can use the current master version of the LTS chart or wait a little until this fix is released in order to resolve this error.
